rllib/env/env_runner.py

```diff
@@ -11,7 +11,7 @@
 tf1, _, _ = try_import_tf()
 
 
-@ExperimentalAPI
+@OldAPIStack
```

Comment:
Why old stack? Isnt't this the class `SingleAgentEnvRunner` and `MultiAgentEnvRunner` inherit from?